python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -3555,3 +3555,42 @@ def convert_broadcast_like(node, **kwargs):
     ]
 
     return nodes
+
+
+@mx_op.register('_contrib_ROIAlign')
+def convert_contrib_roialign(node, **kwargs):
+    """Map MXNet's _contrib_ROIAlign
+    """
+    from onnx.helper import make_node
+    from onnx import TensorProto
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    pooled_size = convert_string_to_list(str(attrs.get('pooled_size')))
+    spatial_scale = float(attrs.get('spatial_scale'))
+    sample_ratio = int(attrs.get('sample_ratio', '0'))
+    position_sensitive = attrs.get('position_sensitive', 'False')
+    aligned = attrs.get('aligned', 'False')
+
+    '''
```

Comment:
Should these checks be commented out?

Reply:
I realized the issue after I created the pr. I have uncommitted them now : )